### DISCUSSION TITLE
Chocked Packets and Max angle data

### DISCUSSION THREAD

**Discussion by wirekurosastak** [danielkrupinski/Osiris#2689] [General]
![image](https://user-images.githubusercontent.com/75095332/110848849-d4244800-82ae-11eb-949b-32cfa0f0a787.png)

What do these features do? And how is it useful?

**Comment by linuxer9** [comment]
chocked packets is the fakelag basically it renders you hard to hit by enemies as you somewhat teleporting as if you are lagging.
max angle delta is like the maximum fov for your aimbot to work with each tick, the default is 255 which is 360 degrees, set this value down to avoid getting VAC auth error as your aimbot will do non-human flicks from vac prespective, TL; DR set it to 30 if you want to rage in casual or DM modes.

**Reply by wirekurosastak** [reply]
At the chocked packets feature should I hold the hotkey or it's toggled?

**Reply by linuxer9** [reply]
> 
> 
> At the chocked packets feature should I hold the hotkey or it's toggled?

yes, set it to a key and hold it when you peek your enemies
